### ISSUE TITLE
Versions do not sort according to their numerical order

### ISSUE THREAD

**Issue by ChrisRackauckas** [JuliaDocs/Documenter.jl#476]
![capture](https://cloud.githubusercontent.com/assets/1814174/25769839/2e6a0e24-31d9-11e7-8a34-997067c4dd6a.PNG)

**Comment by mortenpi** [comment]
Yup, I noticed this as well at one point, but forgot about it. Thanks for opening an issue for this.
